src/current/v23.1/differences-in-metrics-between-third-party-monitoring-integrations-and-db-console.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Differences in Metrics between Third-Party Monitoring Integrations and DB Console
+summary: Learn how metrics can differ between third-party monitoring tools that are integrated with CockroachDB and the DB Console of CockroachDB.
+toc: true
+---
+
+When using [Third-Party Monitoring Integrations]({% link {{ page.version.version }}/third-party-monitoring-tools.md %}), such as the [metrics export feature](https://www.cockroachlabs.com/docs/cockroachcloud/export-metrics), discrepancies may be seen when comparing those metrics charts to ones found on the [Metrics dashboards]({% link {{ page.version.version }}/ui-overview.md %}#metrics) or [custom charts]({% link {{ page.version.version }}/ui-custom-chart-debug-page.md %}) of the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}). This page explains why these different systems may yield different results.
+
+## CockroachDB’s Timeseries Database
+
+CockroachDB stores metrics data in its own internal timeseries database (TSDB). While CockroachDB exposes [point-in-time metrics]({% link {{ page.version.version }}/monitor-cockroachdb-with-prometheus.md %}) via its [Prometheus endpoint]({% link {{ page.version.version }}/monitoring-and-alerting.md %}#prometheus-endpoint),  `/_status/vars`, it also periodically scrapes and writes this information to its own timeseries database storage. This data is scraped every 10 seconds and stored at that resolution. After a period of time determined by the [`timeseries.storage.resolution_10s.ttl` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-timeseries-storage-resolution-10s-ttl), that 10 second resolution data is compacted into a 30 minute resolution. After another period of time determined by the [`timeseries.storage.resolution_30m.ttl cluster setting`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-timeseries-storage-resolution-30m-ttl), the 30 minute resolution data is deleted.
```

Comment:
this info re: the TSDB is great.  i did a search for ['time series database'](https://www.cockroachlabs.com/docs/search?query=time+series+database) and found that this info appears to match what we say in https://www.cockroachlabs.com/docs/v23.1/operational-faqs#why-is-disk-usage-increasing-despite-lack-of-writes - however if there is a subtle difference do you mind filing a followup docs issue? (at a glance it looks the same to me but wanted to get more eyes on it)